### DISCUSSION TITLE
📚 Rename "Repo Handlers" to "Supported Platforms"

### DISCUSSION THREAD

**Discussion by williln** [djangopackages/djangopackages#989] [Docs]
**Is your feature request related to a problem? Please describe.**

There are a couple of sections in the docs that refer to the supported repo sites, and I think it would be helpful to collect that information in one place. This page could also hold the instructions for adding support for a new site. 

The "Repo Handlers" page mostly discusses how to add a new repo platform, but the title "Repo Handler" isn't very descriptive to me. I'm not attached to "Supported Platforms" as a replacement, but I do think it's a bit clearer? There is room for brainstorming, though! 

**Describe the solution you'd like**

Rename the "Repo Handlers" page "Supported Platforms." Add following information: 

- What repo sites are supported? (from Introduction page) 
  - BitBucket (some support)
  - GitHub (fully supported)
  - GitLab (some support)
- Potentially a description of what is NOT supported about BitBucket and GitLab, or what IS supported for GitHub that isn't for the others 

Retain most of the content from https://djangopackages.readthedocs.io/en/latest/repo_handlers.html, with some possible copyedits. 

**Describe alternatives you've considered**

- A new page, but I don't think there are so many handlers that that is warranted 
- A new section, with separate pages per handler. Depending on the level of docs that might need to be added, this could be a solution, but adding this data to the existing page is faster. 

**Additional context**

cc @jefftriplett re: our offline docs discussion.

**Comment by jefftriplett** [comment]
We made a docs change but we should probably define what supported means. GitHub has more features than the other providers, which makes it a better baseline for features.

**Comment by williln** [comment]
@jefftriplett Is it a situation where all three have the same features, and then GH has extra features on top of that? Because that might be a good way to organize it: 

> We support X, Y, and Z platforms. We recommend linking to GitHub if possible. 
> All platforms support the following information: [list of stuff]
> We also support this info for GitHub only: [list of stuff]

Or something like that?

**Comment by jefftriplett** [comment]
@williln yes and all-the-things. 

GitHub has features that others may not have, and the context can also be different because GitHub has more social features. For example, almost all providers will have a notion of forks but stars are fairly unique to GitHub. 

The handlers which manage this are here: https://github.com/djangopackages/djangopackages/tree/main/package/repos

And some tests https://github.com/djangopackages/djangopackages/blob/main/package/tests/test_repos_github.py
